content/post/stash-v2022.12.11/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introducing Stash v2022.12.11
+date: "2022-12-11"
+weight: 10
+authors:
+- Piyush Kanti Das
+tags:
+- backup
+- kubernetes
+- restore
+- stash
+- postgres
+- vault
+---
+
+We are very excited to announce Stash `v2022.12.11`. It comes with a couple of exciting new features, a few bug fixes, and improvements to the codebase. In this post, we are going to highlight the most significant changes.
+
+You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2022.12.11/README.md).
+
+### New Features
+
+Here, are the major features that have been added in this release.
+
+#### Introducing Vault addon
+
+In this release, we are introducing [Hashicorp Vault](https://github.com/stashed/vault) addon. You can now backup and recover your Hashicorp Vault using Stash Enterprise.
+
+#### Add Postgres 15.1 addon
+
+We have added an addon for Postgres 15.1. Now, you should be able to back up your Postgres 15.x.x using this addon.
+
+### Bug Fixes
+
+- Fixed passing PodRuntimeSettings to backup/Restore job ([#1489](https://github.com/stashed/stash/pull/1489)).
```

Comment:
`backup/Restore`  should be replaced with either `backup/restore` or `Backup/Restore`